frontend/src/metabase/visualizations/visualizations/PieChart.jsx

```diff
@@ -281,20 +281,19 @@ export default class PieChart extends Component {
       slices.map(s => s.percentage),
       { style: "percent", maximumSignificantDigits: 3 },
     );
-    const formatPercent = (percent, jsx = true) =>
+    const formatPercent = percent =>
       formatValue(percent, {
-        ...settings.column(cols[metricIndex]),
-        jsx,
+        column: cols[metricIndex],
```

Comment:
Does it even make sense to include `column`?

Reply:
I left that in because `formatValueRaw` always JSON.stringifies if there's no column type to determine what formatting function to call.

I guess I could just pass a fake numeric column, but that feels worse. Maybe we could update [this line](https://github.com/metabase/metabase/blob/fix-10520/frontend/src/metabase/lib/formatting.js#L714) to be `||` rather than `&&`?